src/cobald/monitor/format_line.py

```diff
@@ -16,16 +30,18 @@ def line_protocol(
     :param fields: measurements of the report
     :param timestamp: when the measurement was taken, in **seconds** since the epoch
     """
-    output_str = name
+    _escape_key = escape_key
+    _escape_field = escape_field
+    output_str = name.replace(r',', r'\,').replace(r' ', r'\ ')
```

Comment:
This code is somewhat duplicating the code from function `escape_key`.

Reply:
It is similar, but keys must also escape ``=``. Since the name is only ever replaced once, I did not see a need to define a function for it.